hyf-core-values.md

```diff
@@ -2,22 +2,23 @@
 
 ## Learning
 * Our program is about problem solving and learning how to learn.
-* Take responsibility for your own learning process.
-* Tell us and your fellow students what works for you and what doesn't when it comes to learning.
-* Be active and communicate about things that are not clear.
-* Help us and our teachers improve by giving regular feedback.
+* Take responsibility for your own learning process, what you put into this program is what you get out.
+* Learning how to code is hard. Don't pretend to understand things if you don't, just ask questions. We love questions. 
+* Share your knowledge. Share your favourite resources that helped you understand things/make your life easier. 
+* Don't compare yourself to your fellow students, compare yourself to who you were yesterday.
 * There is a lot of room for struggling/not understanding, there is no room for people who do nothing.
 
 ## Community
-* Volunteers, they are here on their free Sundays, while they could be with their families and friends. Respect their time. Value their time. Give them energy and motivate them. 
```

Comment:
also the part of giving teachers energy i would like to keep

Reply:
I suggest adding: "Show them with your attendance and homework that you respect and value their time."

Reply:
or something like that